build.gradle

```diff
@@ -317,6 +305,12 @@ processResources.dependsOn copyIconsIntoResources
 distZip.exclude("*.x*.exe")
 
 
+idea {
+    project.settings.delegateActions {
+        delegateBuildRunToGradle = false
```

Comment:
Why do we want this?

Reply:
That corresponds to the "build and run using" setting: https://www.jetbrains.com/help/idea/2020.2/gradle-settings.html

> If you have a pure Java or a Kotlin project, it is sometimes better to select IntelliJ IDEA for building a project. IntelliJ IDEA supports the incremental build which significantly speeds up the building process. Yet, keep in mind that the IntelliJ IDEA compiler does not support some parts of the Gradle project build processing and might cause problems in building your project correctly. 

We have it set this way for Terasology.

though gradle has been working on optimizations recently, I wonder if gradle is as fast as IntelliJ's incremental build if we turn on gradle's build cache and filesystem watching. :thinking:

Reply:
I dislike that we do something different in the IDE than on the command line/in CI.

I like that this can speed up the build process and make for a smoother development experience.

I'm not sure whether this should be the default or not. I currently don't expect too many problems occur due to this, but we have to keep this in mind when we run into the typical "it works when run from IntelliJ" issue... 
A part of me wants to throw some Bazel magic at our repos if multi-module build speed is something we are interested in...